tensorflow_addons/seq2seq/loss.py

```diff
@@ -168,6 +182,11 @@ def __init__(self,
         self.sum_over_batch = sum_over_batch
         self.softmax_loss_function = softmax_loss_function
 
+        # Delete the reduction attribute to inform Keras that it
+        # should call this class by the __call__(...) method.
+        if 'reduction' in dir(self):
```

Comment:
`hasattr` seems more appropriate here?

Reply:
Ops, I forget that. Thanks for the comment. it's now fixed in the next commit.